### PR TITLE
Lesson 5 1

### PR DESCRIPTION
**Pull request by BThomas22tech**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -61,8 +61,40 @@ function App() {
         throw new Error(message);
       }
       const airtableData = await response.json();
+      const sortAscending = (objectA, objectB) => {
```
Comment (riousenkai) [inline src/App.js]: - Why are both sorts called back-to-back?
- These sorts aren't doing anything. Why are they placed in the post function?

Comment (BThomas22tech) [reply src/App.js]: This one I didn't quite understand where they need to be to work. What would be the best way to correct this?

Comment (BThomas22tech) [reply src/App.js]: I think i figured some of it out!

File: src/App.js [hunk 2]
```diff
@@ -68,6 +104,22 @@ function App() {
     }
   };
 
+  // const sortAscending = (objectA, objectB) => {
```
Comment (riousenkai) [inline src/App.js]: Remove this function

File: src/App.js [hunk 3]
```diff
@@ -32,6 +32,42 @@ function App() {
         };
         return newTodo;
       });
+
+
+      // const sortAscending = (objectA, objectB) => {
```
Comment (riousenkai) [inline src/App.js]: remove this function since we aren't using it

**Review by riousenkai — APPROVED** [review]
Please resolve my comments before merging

**Pull request merged**